apps/src/templates/census/CensusMap.jsx

```diff
@@ -24,29 +24,29 @@ class CensusMapInfoWindow extends Component {
       case 'YES':
       case 'Y':
         censusMessage = 'We believe this school offers Computer Science.';
-        color = 'green';
+        color = 'legend-offers-cs';
```

Comment:
This reads a bit funky now that it  refers to a set of styles rather than a simple color- would it be worth renaming 'color' here?